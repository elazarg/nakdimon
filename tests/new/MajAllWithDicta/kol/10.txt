אַגְרָה מוּפְחֶתֶת בִּתְבִיעָה בְּשֶׁל עֲבֵירוֹת מִין הַקְדָּמָה: נִפְגְּעוֹת וְנִפְגְּעֵי עֲבֵירוֹת מִין זַכָּאִים לְתַשְׁלוּם אַגְרָה מוּפְחֶתֶת בְּעֵת הַגָּשַׁת תְּבִיעָה בְּגִין נְזָקִים בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם, בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי אוֹ בְּבֵית הַדִּין לַעֲבוֹדָה. בֵּית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה קָבַע כִּי יֵשׁ לְהָחִיל אֶת הָאַגְרָה המופחתת גַּם בִּתְבִיעוֹת המוגשות בְּבֵית מִשְׁפָּט זֶה. לְמֵידָע נוֹסָף רָאוּ הוֹדָעַת בָּתֵּי הַמִּשְׁפָּט - אִגְּרוֹת, אִגְּרוֹת וּשְׂכַר טִרְחָה בְּבָתֵּי הַדִּין לַעֲבוֹדָה וְכֵן תַּקָּנָה 5א לְתַקָּנוֹת בָּתֵּי מִשְׁפָּט (אִגְּרוֹת) וְתִקְּנָהּ 6א לְתַקָּנוֹת בֵּית הַדִּין לַעֲבוֹדָה (אִגְּרוֹת). נִפְגְּעוֹת וְנִפְגְּעֵי עֲבֵירוֹת מִין הַמַּגִּישִׁים תְּבִיעָה בְּשֶׁל נזקיהם זַכָּאִים לְתַשְׁלוּם אִגֶּרֶת תְּבִיעָה מוּפְחֶתֶת. סְכוּמֵי הָאִגְּרוֹת נָכוֹן לִשְׁנַת 2020: -בִּתְבִיעָה המוגשת בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם תשולם אַגְרָה בְּסַךְ 502 ש"ח. -בִּתְבִיעָה המוגשת בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי תשולם אַגְרָה בְּסַךְ 904 ש"ח. -בִּתְבִיעָה המוגשת בְּבֵית הַדִּין לַעֲבוֹדָה תשולם אַגְרָה בְּסַךְ 143 ש"ח. מִי זַכָּאי? -נִפְגַּעַת עֲבֵירַת מִין המגישה אַחַת מִסּוּגֵי הַתְּבִיעוֹת הַבָּאוֹת: -תְּבִיעָה הַנִּגְרֶרֶת לְהַרְשָׁעָה בִּפְלִילִים (תְּבִיעָה אֶזְרָחִית שמוגשת בְּסָמוּךְ לְאַחַר הָרִשְׁעָה ונסמכת עַל הַהַרְשָׁעָה) בַּעֲבֵירוֹת מִין לְפִי חוֹק הָעוֹנָשִׁין -תְּבִיעָה שֶׁהִגִּישׁ הַנִּפְגָּע נֶגֶד מִי שֶׁבִּיצֵּעַ מַעֲשֶׂה הַמְּהַוֶּוה עֲבֵירַת מִין לְפִי חוֹק הָעוֹנָשִׁין. -תְּבִיעָה לְפִי חוֹק לִמְנִיעַת הַטְרָדָה מִינִית. תַּהֲלִיךְ מִימּוּשׁ הַזְּכוּת: -כַּאֲשֶׁר הַתְּבִיעָה מוּגֶּשֶׁת לְבֵית מִשְׁפָּט הַשָּׁלוֹם, בֵּית הַמִּשְׁפָּט הַמְּחוֹזִי אוֹ בֵּית הַדִּין לַעֲבוֹדָה, הַזְּכוּת נִיתֶּנֶת בְּאוֹפֶן אוֹטוֹמָטִי. בְּעֵת הַגָּשַׁת הַתְּבִיעָה יֵשׁ לְשַׁלַּם אֶת הָאַגְרָה המופחתת. -כַּאֲשֶׁר הַתְּבִיעָה מוּגֶּשֶׁת לְבֵית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה, יֵשׁ לְהַגִּישׁ בַּקָּשָׁה לְתַשְׁלוּם אַגְרָה מוּפְחֶתֶת. -בְּכָל מִקְרֶה אִם הַתְּבִיעָה לֹא מוּגֶּשֶׁת רַק בְּשֶׁל נְזָקִים שֶׁקְּשׁוּרִים לעבירת מִין, בֵּית הַמִּשְׁפָּט עָשׂוּי לְפַצֵּל אֶת הָאַגְרָה, ולהחיל אֶת הָאַגְרָה המופחתת רַק עַל חֵלֶק מֵהַתְּבִיעָה שֶׁמִּתְיַיחֵס לִנְזָקִים שֶׁגָּרְמָה עֲבֵירַת הַמִּין. חָשׁוּב לָדַעַת: -קַיָּימוֹת תַּקָּנוֹת הקובעות אִגְּרוֹת מופחתות עֲבוּר הַגָּשַׁת תְּבִיעוֹת בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם, בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי וּבְבֵית הַדִּין לַעֲבוֹדָה בִּלְבָד, אוּלָם פָּסַק דִּין שֶׁל בֵּית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה קָבַע כִּי הַהֶסְדֵּר שֶׁל אַגְרָה מוּפְחֶתֶת יָחוּל גַּם בִּתְבִיעוֹת עַל עֲבֵירוֹת מִין המוגשות בְּבֵית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה.