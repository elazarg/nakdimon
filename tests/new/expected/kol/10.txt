אַגְרָה מוּפְחֶתֶת בִּתְבִיעָה בְּשֶׁל עֲבֵירוֹת מִין
הַקְדָּמָה:
נִפְגָּעוֹת וְנִפְגְּעֵי עֲבֵירוֹת מִין זַכָּאִים לְתַשְׁלוּם אַגְרָה מוּפְחֶתֶת בְּעֵת הַגָּשַׁת תְּבִיעָה בְּגִין נְזָקִים בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם, בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי אוֹ בְּבֵית הַדִּין לַעֲבוֹדָה.
בֵּית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה קָבַע כִּי יֵשׁ לְהָחִיל אֶת הָאַגְרָה הַמּוּפְחֶתֶת גַּם בִּתְבִיעוֹת הַמּוּגָּשׁוֹת בְּבֵית מִשְׁפָּט זֶה.
לְמֵידָע נוֹסָף רָאוּ הוֹדָעַת בָּתֵּי הַמִּשְׁפָּט - אֲגָרוֹת, אֲגָרוֹת וּשְׂכַר טִרְחָה בְּבָתֵּי הַדִּין לַעֲבוֹדָה וְכֵן תַּקָּנָה 5א לְתַקָּנוֹת בָּתֵּי מִשְׁפָּט (אֲגָרוֹת) וְתַקָּנָה 6א לְתַקָּנוֹת בֵּית הַדִּין לַעֲבוֹדָה (אֲגָרוֹת).
נִפְגָּעוֹת וְנִפְגְּעֵי עֲבֵירוֹת מִין הַמַּגִּישִׁים תְּבִיעָה בְּשֶׁל נִזְקֵיהֶם זַכָּאִים לְתַשְׁלוּם אַגְרַת תְּבִיעָה מוּפְחֶתֶת.
סְכוּמֵי הָאֲגָרוֹת נָכוֹן לִשְׁנַת 2020:
-בִּתְבִיעָה הַמּוּגֶּשֶׁת בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם תְּשׁוּלַּם אַגְרָה בְּסַךְ 502 ש"ח.
-בִּתְבִיעָה הַמּוּגֶּשֶׁת בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי תְּשׁוּלַּם אַגְרָה בְּסַךְ 904 ש"ח.
-בִּתְבִיעָה הַמּוּגֶּשֶׁת בְּבֵית הַדִּין לָעֲבוֹדָה תְּשׁוּלַּם אַגְרָה בְּסַךְ 143 ש"ח.
מִי זַכַּאי?
-נִפְגַּעַת עֲבֵירַת מִין הַמַּגִּישָׁה אַחַת מִסּוּגֵי הַתְּבִיעוֹת הַבָּאוֹת:
-תְּבִיעָה הַנִּגְרֶרֶת לְהַרְשָׁעָה בִּפְלִילִים (תְּבִיעָה אֶזְרָחִית שֶׁמּוּגֶּשֶׁת בְּסָמוּךְ לְאַחַר הַרְשָׁעָה וְנִסְמֶכֶת עַל הַהַרְשָׁעָה) בַּעֲבֵירוֹת מִין לְפִי חוֹק הָעוֹנְשִׁין
-תְּבִיעָה שֶׁהִגִּישׁ הַנִּפְגָּע נֶגֶד מִי שֶׁבִּיצֵּעַ מַעֲשֶׂה הַמְּהַוֶּוה עֲבֵירַת מִין לְפִי חוֹק הָעוֹנְשִׁין.
-תְּבִיעָה לְפִי חוֹק לִמְנִיעַת הַטְרָדָה מִינִית.
תַּהֲלִיךְ מִימּוּשׁ הַזְּכוּת:
-כַּאֲשֶׁר הַתְּבִיעָה מוּגֶּשֶׁת לְבֵית מִשְׁפַּט הַשָּׁלוֹם, בֵּית הַמִּשְׁפָּט הַמְּחוֹזִי אוֹ בֵּית הַדִּין לַעֲבוֹדָה, הַזְּכוּת נִיתֶּנֶת בְּאוֹפֶן אוֹטוֹמָטִי. בְּעֵת הַגָּשַׁת הַתְּבִיעָה יֵשׁ לְשַׁלֵּם אֶת הָאַגְרָה הַמּוּפְחֶתֶת.
-כַּאֲשֶׁר הַתְּבִיעָה מוּגֶּשֶׁת לְבֵית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה, יֵשׁ לְהַגִּישׁ בַּקָּשָׁה לְתַשְׁלוּם אַגְרָה מוּפְחֶתֶת.
-בְּכָל מִקְרֶה אִם הַתְּבִיעָה לֹא מוּגֶּשֶׁת רַק בְּשֶׁל נְזָקִים שֶׁקְּשׁוּרִים לַעֲבֵירַת מִין, בֵּית הַמִּשְׁפָּט עָשׂוּי לְפַצֵּל אֶת הָאַגְרָה, וּלְהָחִיל אֶת הָאַגְרָה הַמּוּפְחֶתֶת רַק עַל חֵלֶק מֵהַתְּבִיעָה שֶׁמִּתְיַיחֵס לַנְּזָקִים שֶׁגָּרְמָה עֲבֵירַת הַמִּין.
חָשׁוּב לָדַעַת:
-קַיָּימוֹת תַּקָּנוֹת הַקּוֹבְעוֹת אֲגָרוֹת מוּפְחָתוֹת עֲבוּר הַגָּשַׁת תְּבִיעוֹת בְּבֵית הַמִּשְׁפָּט הַשָּׁלוֹם, בְּבֵית הַמִּשְׁפָּט הַמְּחוֹזִי וּבְבֵית הַדִּין לַעֲבוֹדָה בִּלְבַד, אוּלָם פְּסַק דִּין שֶׁל בֵּית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה קָבַע כִּי הַהֶסְדֵּר שֶׁל אַגְרָה מוּפְחֶתֶת יָחוּל גַּם בִּתְבִיעוֹת עַל עֲבֵירוֹת מִין הַמּוּגָּשׁוֹת בְּבֵית הַמִּשְׁפָּט לְעִנְיְינֵי מִשְׁפָּחָה.
